docs/man_pages/lib-management/plugin-add.md

```diff
@@ -43,15 +45,16 @@ Enables <%=plugins%> for your project. <% if(isHtml) { %>If the plugin has plugi
 <% if((isConsole && (isCordova || isNativeScript)) || isHtml) { %>
 ### Options
 * `--available` - Lists all <%=plugins%> that you can enable in your project and shows information about their variables, if any.
-* `--var.<Variable ID>` - Sets the value for the specified plugin variable in all configurations.
-	<% if(isHtml) { %><br /><% } %><% if(isCordova) { %>(Apache Cordova-only) If `--debug` or `--release` is specified, sets the variable for the respective configuration of the hybrid project.<% } %>
+* `--var.<Variable ID>` - Sets the value for the specified plugin variable in all configurations.<% if(isHtml) { %><br /><% } %><% if(isCordova) { %>(Apache Cordova-only) If `--debug` or `--release` is specified, sets the variable for the respective configuration of the hybrid project.<% } %>
 <% if(isCordova) {%>* `--latest` - Enables the latest version of the specified Apache Cordova plugin.<% if(isHtml) { %> This option is applicable only to Apache Cordova projects.<% } %>
 * `--default` - Enables the default version of the specified Apache Cordova plugin.<% if(isHtml) { %> This option is applicable only to Apache Cordova projects.<% } %>
 * `--debug` - Enables the specified Apache Cordova plugin for the Debug build configuration only. If `--available` is set, lists all plugins that you can enable for the Debug build configuration.<% if(isHtml) { %> This option is applicable only to Apache Cordova projects.<% } %>
 * `--release` - Enables the specified Apache Cordova plugin for the Release build configuration only. If `--available` is set, lists all plugins that you can enable for the Release build configuration.<% if(isHtml) { %> This option is applicable only to Apache Cordova projects.<% } %>
 * `--var.debug.<Variable ID>` - Sets the value for the specified Apache Cordova plugin variable for the Debug configuration only.<% if(isHtml) { %> This option is applicable only to Apache Cordova projects.<% } %>
 * `--var.release.<Variable ID>` - Sets the value for the specified Apache Cordova plugin variable for the Release configuration only.<% if(isHtml) { %> This option is applicable only to Apache Cordova projects.<% } %><% } %>
-<% if(isNativeScript) { %>* `--count` - When set along with `--available`, specifies the number of npm and NativeScript modules that will be listed.<% if(isHtml) { %> This option is applicable only to NativeScript projects.<% } %><% } %>
+<% if(isNativeScript) { %>* `--count` - When set along with `--available`, specifies the number of npm and NativeScript modules that will be listed.<% if(isHtml) { %> This option is applicable only to NativeScript projects.<% } %>
```

Comment:
<% if(isNativeScript) { %>* `--count` - If `--available` is set, specifies the number of npm and NativeScript modules that will be listed.<% if(isHtml) { %> This option is applicable only to NativeScript projects.<% } %>